test/test_torch.py

```diff
@@ -1667,6 +1667,59 @@ def test_horovod_join_broadcast(self):
                 ret = hvd.join(hvd.local_rank())
             else:
                 ret = hvd.join()
+    
+    def test_horovod_sync_batch_norm(self):
```

Comment:
May also want to include a skip if Python version is < 3 (integration tests passing because we have no Python 2 GPU tests anymore).

Reply:
Added skip, but we should deprecate all Py2 tests :-)

Reply:
Agreed.  Tentative plan is to drop Python 2 in 0.20.0.